### PR TITLE
Implement tabbedview for Workspace + Revert some conditions implemented in the dossier type for the workspace

### PR DESCRIPTION
**Pull request by maethu**
- Add a minimal tabbedview integration for the Workspace based on the Dossier tabbedview
- Make task addable
- Add Workspace specific sequence and `namefromtitle` behavior
- Revert Workspace specific implementation in opengever.dossier

### CODE REVIEW

File: opengever/core/upgrades/20171213174551_add_workspace_content_type/types/opengever.workspace.workspace.xml [hunk 1]
```diff
@@ -36,11 +36,12 @@
   </property>
 
   <!-- View information -->
-  <property name="immediate_view">view</property>
-  <property name="default_view">view</property>
+  <property name="immediate_view">tabbed_view</property>
+  <property name="default_view">tabbed_view</property>
   <property name="default_view_fallback">False</property>
   <property name="view_methods">
     <element value="view" />
```
Comment (jone) [inline opengever/core/upgrades/20171213174551_add_workspace_content_type/types/opengever.workspace.workspace.xml]: ✂️ ?

File: opengever/core/upgrades/20171220143920_configure_tabbedview_as_default_view_for_workspace/types/opengever.workspace.workspace.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n" name="opengever.workspace.workspace" meta_type="Dexterity FTI" i18n:domain="opengever.core">
+
+  <!-- View information -->
+  <property name="immediate_view">tabbed_view</property>
+  <property name="default_view">tabbed_view</property>
+  <property name="default_view_fallback">False</property>
+  <property name="view_methods">
+    <element value="view" />
```
Comment (jone) [inline opengever/core/upgrades/20171220143920_configure_tabbedview_as_default_view_for_workspace/types/opengever.workspace.workspace.xml]: ✂️ ?

File: opengever/workspace/behaviors/namefromtitle.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+from zope.interface import implements
+
+
+class IWorkspaceNameFromTitle(INameFromTitle):
+    """Behavior interface.
+    """
+
+
+class WorkspaceNameFromTitle(dossiernamefromtitle.DossierNameFromTitle):
+    """Speical name from title behavior for letting the normalizing name
```
Comment (jone) [inline opengever/workspace/behaviors/namefromtitle.py]: `Speical`

File: opengever/workspace/browser/tabbed.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
+from opengever.dossier import _
+from opengever.tabbedview import GeverTabbedView
+
+
+class WorkspaceTabbedView(GeverTabbedView):
+    """Define the tabs available on a Dossier."""
```
Comment (jone) [inline opengever/workspace/browser/tabbed.py]: ✂️ (not "just" a dossier)

**Comment by maethu** [comment]
@jone updated, thx!!

**Pull request merged**